### PR TITLE
vagrant: add support for Linode VMs

### PR DESCRIPTION
**Pull request by batrick**
This will be used in upcoming performance testing.

### CODE REVIEW

File: vagrant_variables.yml.linode [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+cloud_distribution: 'CentOS 7'
+cloud_datacenter: 'newark'
+
+# Memory for each Linode instance, this determines price! See Linode plans.
+memory: 2048
+
+# The private network on Linode, you probably don't want to change this.
+subnet: 192.168.0
+
+# The interface the monitors should listen on, this is currently ignored for linode.
```
Comment (leseb) [inline vagrant_variables.yml.linode]: Why don't we simply remove this then?

File: vagrant_variables.yml.linode [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+# The sync directory changes based on vagrant box
+# Set to /home/vagrant/sync for Centos/7, /home/{ user }/vagrant for openstack and defaults to /vagrant
+# vagrant_sync_dir: /home/vagrant/sync
+
+os_tuning_params:
+  - { name: kernel.pid_max, value: 4194303 }
+  - { name: fs.file-max, value: 26234859 }
+
+ceph_conf_overrides:
```
Comment (leseb) [inline vagrant_variables.yml.linode]: is it a left over?

File: vagrant_variables.yml.linode (hunk 1 above)
Comment (batrick) [inline vagrant_variables.yml.linode]: Okay, will do.

File: vagrant_variables.yml.linode (hunk 2 above)
Comment (batrick) [inline vagrant_variables.yml.linode]: Not really. I thought it'd be useful to have an example `vagrant_variables.yml.X` with the new ceph_conf_overrides support. I will remove the `mds debug` but I'd like to keep the `fuse default permissions` override (as it's generally a large performance improvement for cephfs).

Comment (leseb) [inline vagrant_variables.yml.linode]: Hum `ceph_conf_overrides` can be used and should from `group_vars/all`, so I'd like to keep it that way. When it comes to optimizations, I have a local branch for "profiles". These use `ceph_conf_overrides` and provide some guidance on how to tune/tweak specific deployment. The fuse option falls down into this so it's a good candidate once I push the branch, until then I suggest we don't do anything.

Comment (batrick) [inline vagrant_variables.yml.linode]: Oh, that makes a lot more sense. Thanks for clarifying! I will remove it.

Comment (leseb) [inline vagrant_variables.yml.linode]: Np, thanks!

**Pull request merged**